src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+xref: OMIA:001371 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001371"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001371"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009370 {source="OMIA:001371"} ! cross-species analog L-2-hydroxyglutaric aciduria
+
+[Term]
+id: MONDO:1010245
+name: hyperekplexia, non-human animal
+def: "Hyperekplexia that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001594 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001594"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001594"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0013827 {source="OMIA:001594"} ! cross-species analog hyperekplexia 3
+
+[Term]
+id: MONDO:1010247
+name: lissencephaly and cerebellar hypoplasia, non-human animal
+def: "Lissencephaly and cerebellar hypoplasia that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001867 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001867"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001867"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009760 {source="OMIA:001867"} ! cross-species analog Norman-Roberts syndrome
+
+[Term]
+id: MONDO:1010249
+name: ataxia telangiectasia, non-human animal
+def: "Ataxia telangiectasia that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:002044 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002044"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:002044"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0008840 {source="OMIA:002044"} ! cross-species analog ataxia telangiectasia
+
+[Term]
+id: MONDO:1010252
+name: succinic semialdehyde dehydrogenase deficiency, non-human animal
+def: "Succinic semialdehyde dehydrogenase deficiency that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:002250 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002250"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:002250"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0010083 {source="OMIA:002250"} ! cross-species analog succinic semialdehyde dehydrogenase deficiency
+
+[Term]
+id: MONDO:1010255
+name: feather color albinism, sex-linked, imperfect, non-human animal
+def: "Feather color albinism, sex-linked, imperfect that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000370 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000370"} ! non-human animal disease
+is_a: MONDO:1011328 {source="OMIA:000370", source="https://orcid.org/0000-0002-5002-8648"} ! integumentary system disorder, non-human animal
+
+[Term]
+id: MONDO:1010256
+name: coat color albinism, oculocutaneous type IV, non-human animal
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.